Extend the notes about noexec option for mounting /tmp and /var/tmp (#1677)
uyuni-project/uyuni-docs: modules/client-configuration/pages/contact-methods-saltbundle.adoc

For more information, see xref:client-configuration:bootstrap-repository.adoc#_prepare_to_create_a_bootstrap_repository[Prepare to Create a Bootstrap Repository]
====

[IMPORTANT]
====

> Salt SSH is using [path]``/var/tmp`` to deploy Salt Bundle to and execute the Salt on the client with the bundled Python.

Reviewer: ```suggestion
Salt SSH is using [path]``/var/tmp`` to deploy Salt Bundle to and execute Salt commands on the client with the bundled Python.
```

Therefore you must not mount [path]``/var/tmp`` with the [option]``noexec`` option.

> It is not possible to bootstrap the clients, which have [path]``/var/tmp`` mounted with [option]``noexec`` option, via {webui} as the bootstrap process is using Salt SSH to reach a client.

Reviewer: ```suggestion
It is not possible to bootstrap the clients, which have [path]``/var/tmp`` mounted with [option]``noexec`` option, with the {webui} because the bootstrap process is using Salt SSH to reach a client.
```
